scripts.js

```diff
@@ -1 +1,89 @@
-console.log('Test');
\ No newline at end of file
+// First task:
+// Add up all of the salaries
+const salaries = {
+    john: 50,
+    adam: 100,
+    katie: 75
+}
+const salarySum = salaries.john + salaries.adam + salaries.katie;
+console.log(salarySum);
+
+// Second task:
+// Write the isObject() function. Use typeof operator.
+function isObject(valueToCheck) {
+    return typeof valueToCheck === "object" && valueToCheck !== null;
+}
+
+// Third task:
+// Write the removeProperty function.
+// It should return true  if there is a property with a given key to remove.
+// Otherwise, it should return false.
+const user = {
+    name: 'John',
+}
+function removeProperty(object, key) {
+    if(key in object) {
+        delete object[key]
+        return true;
+    } else return false;
+}
+
+// Fourth task
+// Write the checkIfUsersHaveTheSameName function.
+
+const firstJohn = {
+    firstName: 'John',
+    lastName: 'Smith'
+}
+
+const secondJohn = {
+    firstName: 'John',
+    lastName: 'Smith'
+}
+
+const adam = {
+    firstName: 'Adam',
+    lastName: 'Johnson'
+}
+
+function checkIfUsersHaveTheSameName(objectOneName, objectTwoName) {
+    return objectOneName.firstName === objectTwoName.firstName;
```

Comment:
`objectOneName` is not a good argument name. Please note that the function is named `checkIfUsersHaveTheSameName`, so it suggests that we can expect users in the arguments.